language-server/modules/langserver-core/src/test/resources/packages/metadata/project-services_expected.json

```diff
@@ -1,4 +1,5 @@
 {
+  "orgName": "bob",
```

Comment:
Dont we need to add a test for a scenario with orgName key does not exists

Reply:
When `orgName` is not specified in `Ballerina.toml`, the `ProjectAPI` defaults to using the local machine's username as the organization name. Consequently, when the test environment changes, different `orgName` values are generated if `orgName` is absent in `Ballerina.toml`. This inconsistency makes it hard to test this.